### ISSUE TITLE
After update to Angular 8, webpack won't process lazy loaded routes

### ISSUE THREAD

**Issue by muhammedMoussa** [angular/angular#31837]
Followed the update instructions on https://update.angular.io/#7.0:8.0. When i try to serve or build the application this error occurs, haven't been able to find any similar issues.
Any help is appreciated.

```
ERROR in ./src/app/app-routing.module.ts 17:28
Module parse failed: Unexpected token (17:28)
File was processed with these loaders:
 * ./node_modules/@ngtools/webpack/src/index.js
You may need an additional loader to handle the result of these loaders.
|     {
|         path: 'equipment',
>         loadChildren: () => import('./modules/equipment/equipment.module').then(m => m.EquipmentModule),
|         canActivate: [RequireAuthenticatedUserRouteGuardService, CanActivateRouteGuard],
|         data: { ModuleId: 428 }
```

package.json

```
{
  "name": "concrete",
  "version": "0.0.0",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e",
    "stage": "ng build --prod --base-href /Concrete/"
  },
  "private": true,
  "dependencies": {
    "@angular/animations": "~8.1.2",
    "@angular/common": "~8.1.2",
    "@angular/compiler": "~8.1.2",
    "@angular/core": "~8.1.2",
    "@angular/forms": "~8.1.2",
    "@angular/platform-browser": "~8.1.2",
    "@angular/platform-browser-dynamic": "~8.1.2",
    "@angular/router": "~8.1.2",
    "@ngrx/core": "^1.2.0",
    "@ngrx/effects": "^7.4.0",
    "@ngrx/store": "^7.4.0",
    "@ngrx/store-devtools": "^7.4.0",
    "@ngx-translate/core": "^11.0.1",
    "@ngx-translate/http-loader": "^4.0.0",
    "angular-auth-oidc-client": "^9.0.7",
    "bootstrap": "^4.3.1",
    "core-js": "^2.6.5",
    "devextreme": "19.1.4",
    "devextreme-angular": "19.1.4",
    "jquery": "^3.4.1",
    "moment": "^2.24.0",
    "oidc-client": "^1.7.1",
    "rxjs": "~6.5.2",
    "stream": "0.0.2",
    "subsink": "^1.0.0",
    "tslib": "^1.9.0",
    "zone.js": "~0.9.1",
    "@aspnet/signalr": "^1.1.2",
    "acorn": "^6.0.5"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.801.2",
    "@angular/cli": "~8.1.2",
    "@angular/compiler-cli": "~8.1.2",
    "@angular/language-service": "~8.1.2",
    "@types/jasmine": "~2.8.8",
    "@types/jasminewd2": "~2.0.3",
    "@types/node": "^8.10.50",
    "codelyzer": "^5.0.1",
    "devextreme-intl": "^18.2.8",
    "jasmine-core": "~2.99.1",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~3.1.1",
    "karma-chrome-launcher": "~2.2.0",
    "karma-coverage-istanbul-reporter": "~2.0.1",
    "karma-jasmine": "~1.1.2",
    "karma-jasmine-html-reporter": "^0.2.2",
    "protractor": "~5.4.0",
    "ts-node": "~7.0.0",
    "tslint": "~5.11.0",
    "typescript": "~3.4.0"
  }
}

```

**Comment by santoshyadav198613** [comment]
Refer https://github.com/angular/angular/issues/31421

**Comment by muhammedMoussa** [comment]
solved by add  `"webpack": "^4.35.3"` ... since import() bug fixed.
